fixtures/test_proxy/variables.tf

```
@@ -0,0 +1,54 @@
+# Secret Manager and Certificate
+# -------------------------
+
+variable "mitmproxy_ca_certificate_secret" {
+  default     = null
+  description = <<-EOD
+  The identifier of a secret comprising a Base64 encoded PEM certificate file for the mitmproxy Certificate Authority.
+  EOD
+  type        = string
+}
+
+variable "mitmproxy_ca_private_key_secret" {
+  default     = null
+  description = <<-EOD
+  The identifier of a secret comprising a Base64 encoded PEM private key file for the mitmproxy Certificate Authority.
+  EOD
+  type        = string
+}
+
+# Network
+# -------
+
+variable "subnet_id" {
+  default     = ""
```

We want to use `null` as opposed to empty strings for defaults.
```suggestion
  default     = null
```